handlers/handler-publickey-auth/src/main/java/org/apache/servicecomb/authentication/provider/AccessController.java

```diff
@@ -86,18 +86,28 @@ private boolean matchFound(Microservice microservice, Map<String, ConfigurationI
     for (ConfigurationItem item : ruleList.values()) {
       // TODO: Currently we only support property, not support tags. And we will support tags later.
       if (ConfigurationItem.CATEGORY_PROPERTY.equals(item.category)) {
-        // TODO: Currently we only support to configure serviceName. And we will support others later.
+        // TODO: Currently we only support to configure serviceName & version. And we will support others later.
         if ("serviceName".equals(item.propertyName)) {
           if (isPatternMatch(microservice.getServiceName(), item.rule)) {
             matched = true;
             break;
           }
         }
+        if ("version".equals(item.propertyName)) {
```

Comment:
Maybe you can write this code using reflection to find all Microservice String getters and match in a general way to avoid many if statements.

Reply:
ok, i will deal with this